main.cpp

```diff
@@ -183,6 +183,7 @@ class Game
         cout << texts.lose;
         Pause();
         win = false;
+        exit(0);
```

Comment:
I tried the code on Linux environment (Ubuntu 20.04.6 LTS) and its working pretty fine. no problem here good work 👍

Merging the PR since no other bugs are there i suppose